src/settings/Settings.tsx

```diff
@@ -389,10 +389,10 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_ACCOUNT_SETTINGS,
         default: null,
     },
-    "feature_hidden_read_receipts": {
-        supportedLevels: LEVELS_FEATURE,
-        displayName: _td("Don't send read receipts"),
-        default: false,
+    "sendReadReceipts": {
```

Comment:
Before we transition this feature out of labs, I'd like to see the MSC enter a reasonably positive FCP cycle. Doesn't have to be written into spec, but would prefer to only have to write the feature once without maintaining backwards compatibility.

Reply:
I definitely agree with that; I've made the PR this early to gather feedback from design early to avoid delays on that